### PR TITLE
Arc

### PR DESCRIPTION
**Pull request by ray-turla**
# Changes

1. arc_ordinal_theta
2. arc_params
3. arc_pie_pyramid
4. arc_radial_histogram
5. arc_radial

### CODE REVIEW

File: src/arc_params.vl.js [hunk 1]
```diff
@@ -1,11 +1,90 @@
 import * as vl from 'vega-lite-api';
+// import { hconcat } from 'vega-lite-api';
```
Comment (mhkeller) [inline src/arc_params.vl.js:2]: Could you remove this commented out line?

File: src/arc_params.vl.js [hunk 2]
```diff
@@ -1,11 +1,90 @@
...
+      "type": "arc",
+      "radius": {"expr": "radius"},
+      "radius2": {"expr": "radius2"},
+      "theta": {"expr": "theta_single_arc"},
+      "theta2": {"expr": "theta2_single_arc"},
+      "cornerRadius": {"expr": "cornerRadius"},
+      "padAngle": {"expr": "padAngle"},
+      "strokeWidth": {"expr": "strokeWidth"}
+    }
+    */
```
Comment (mhkeller) [inline src/arc_params.vl.js:24]: And also remove this comment block

**Comment by mhkeller** [comment]
Thanks a bunch for putting these together! I left two notes to just remove those comments and I'll go ahead and merge this. The rationale for removing the comments is once these are done, it's best if they're all standardized so that they could be put into the online docs as easily as possible. Thanks again!

**Comment by mhkeller** [comment]
Thanks!

**Comment by ray-turla** [comment]
> Thanks a bunch for putting these together! I left two notes to just remove those comments and I'll go ahead and merge this. The rationale for removing the comments is once these are done, it's best if they're all standardized so that they could be put into the online docs as easily as possible. Thanks again!

My bad on that one. I had that over looked. You're welcome. Thanks for letting me contribute. I got more coming!

**Comment by mhkeller** [comment]
Great! One question actually. I just saw in the readme you added a note about the type ordinal field not existing in the spec. But I do see it [here](https://github.com/mhkeller/vega-lite-api-examples/blob/732d29332db509f06c0fda2a167bb869a844c039/specs/arc_pie_pyramid.vl.json#L34)

**Comment by ray-turla** [comment]
> 

I read that part prior to working on arc examples. I could remove it on the next PR.

**Comment by mhkeller** [comment]
Ah nevermind. That was my addition.

**Pull request merged**